account_asset_management/security/ir.model.access.csv

```diff
@@ -13,3 +13,7 @@ access_account_asset_recompute_trigger_manager,account.asset.recompute.trigger,m
 access_account_asset_group_invoice,account.asset.group,model_account_asset_group,account.group_account_invoice,1,0,0,0
 access_account_asset_group_user,account.asset.group,model_account_asset_group,account.group_account_user,1,0,0,0
 access_account_asset_group_manager,account.asset.group,model_account_asset_group,account.group_account_manager,1,1,1,1
+access_account_asset_remove_user,account.asset.remove,model_account_asset_remove,account.group_account_user,1,0,0,0
+access_account_asset_remove_manager,account.asset.remove,model_account_asset_remove,account.group_account_manager,1,1,1,1
+access_account_asset_compute_user,account.asset.compute,model_account_asset_compute,account.group_account_user,1,0,0,0
+access_account_asset_compute_manager,account.asset.compute,model_account_asset_compute,account.group_account_manager,1,1,1,1
```

Comment:
This is not correct, as an accounting user can do the removal process or compute assets. Put only 2 lines with 1,1,1,1 for accounting users (account managers inherit from account users).